Makefile

```diff
@@ -47,16 +59,29 @@ debug:
 		--ignore-operation-annotation=$(IGNORE_OPERATION_ANNOTATION) \
 		--leader-election=$(LEADER_ELECTION)
 
+.PHONY: start-admission
+start-admission:
+	@LEADER_ELECTION_NAMESPACE=garden go run \
+		-ldflags $(LD_FLAGS) \
+		./cmd/$(EXTENSION_PREFIX)-$(ADMISSION_NAME) \
+		--maxAllowedCIDRs=50 \
+		--webhook-config-server-host=0.0.0.0 \
+		--webhook-config-server-port=$(WEBHOOK_CONFIG_PORT) \
+		--webhook-config-mode=$(WEBHOOK_CONFIG_MODE) \
+		$(WEBHOOK_PARAM)
+
 #################################################################
 # Rules related to binary build, Docker image build and release #
 #################################################################
 
 PUSH ?= false
-images: export KO_DOCKER_REPO = $(REPO)
+images: export KO_DOCKER_REPO = $(REPO)/$(EXTENSION_PREFIX)-$(NAME)
 
```

Comment:
I think this is not needed at all, `KO_DOCKER_REPO` is set in the command, this should be enough.
```suggestion
```